flowmachine/flowmachine/features/subscriber/unique_subscribers_from_queries.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+from flowmachine.core.query import Query
+
+
+class UniqueSubscribersFromQueries(Query):
+    """
+    Given a list of queries with a 'subscriber' column, returns a table of unique subscribers.
+    """
+
+    def __init__(self, query_list: List[Query]):
+        self.query_list = query_list
```

Comment:
Perhaps best to convert this to a set when setting the attribute, so that the query ID doesn't depend on the order of queries in the input list.

Reply:
(At least, I'd hope using a set here would have the desired effect. I'm not 100% sure...)

Reply:
Shouldn't matter - they'll get sorted by their query ids by the id builder.

Reply:
Ah, so they will! OK, this is fine as it is, then.